prboom2/src/memio.c

```diff
@@ -198,7 +198,7 @@ int mem_fseek(MEMFILE *stream, signed long position, mem_rel_t whence)
       return -1;
   }
 
-  if (newpos < stream->buflen)
+  if (newpos <= stream->buflen)
```

Comment:
Could you explain this change / what consequences there might be?

Reply:
Without this change SEEK_END file will never work. Same fix in Odamex: https://github.com/odamex/odamex/blob/7190b0217abfd7fb22ad6f9f10026324367e8f3a/common/m_memio.cpp#L200